templates/CRM/Admin/Form/RelationshipType.tpl

```diff
@@ -24,44 +24,4 @@
 *}
 {* this template is used for adding/editing relationship types  *}
-<div class="crm-block crm-form-block crm-relationship-type-form-block">
-      <div class="crm-submit-buttons">{include file="CRM/common/formButtons.tpl" location="top"}</div>
-  {if $action eq 8}
-      <div class="messages status no-popup">
-          <div class="icon inform-icon"></div>
-          {ts}WARNING: Deleting this option will result in the loss of all Relationship records of this type.{/ts} {ts}This may mean the loss of a substantial amount of data, and the action cannot be undone.{/ts} {ts}Do you want to continue?{/ts}
-
-
-      </div>
-     {else}
-      <table class="form-layout-compressed">
-            <tr class="crm-relationship-type-form-block-label_a_b">
-                <td class="label">{$form.label_a_b.label} {if $action == 2}{include file='CRM/Core/I18n/Dialog.tpl' table='civicrm_relationship_type' field='label_a_b' id=$relationship_type_id}{/if}</td>
-                <td>{$form.label_a_b.html}<br />
-                <span class="description">{ts}Label for the relationship from Contact A to Contact B. EXAMPLE: Contact A is 'Parent of' Contact B.{/ts}</span></td>
-            </tr>
-            <tr class="crm-relationship-type-form-block-label_b_a">
-                <td class="label">{$form.label_b_a.label} {if $action == 2}{include file='CRM/Core/I18n/Dialog.tpl' table='civicrm_relationship_type' field='label_b_a' id=$relationship_type_id}{/if}</td>
-                <td>{$form.label_b_a.html}<br />
-                <span class="description">{ts}Label for the relationship from Contact B to Contact A. EXAMPLE: Contact B is 'Child of' Contact A. You may leave this blank for relationships where the name is the same in both directions (e.g. Spouse).{/ts}</span></td>
-            </tr>
-            <tr class="crm-relationship-type-form-block-contact_types_a">
-                <td class="label">{$form.contact_types_a.label}</td>
-                <td>{$form.contact_types_a.html}</td>
-            </tr>
-            <tr class="crm-relationship-type-form-block-contact_types_b">
-                <td class="label">{$form.contact_types_b.label}</td>
-                <td>{$form.contact_types_b.html}</td>
-            </tr>
-            <tr class="crm-relationship-type-form-block-description">
-                <td class="label">{$form.description.label} {if $action == 2}{include file='CRM/Core/I18n/Dialog.tpl' table='civicrm_relationship_type' field='description' id=$relationship_type_id}{/if}</td>
-                <td>{$form.description.html}</td>
-            </tr>
-            <tr class="crm-relationship-type-form-block-is_active">
-                <td class="label">{$form.is_active.label}</td>
-                <td>{$form.is_active.html}</td>
-            </tr>
-        </table>
-    {/if}
-    <div class="crm-submit-buttons">{include file="CRM/common/formButtons.tpl" location="bottom"}</div>
-</div>
+{include file="CRM/Core/Form/EntityForm.tpl"}
```

Comment:
Instead of having a `RelationshipType.tpl` file at all, maybe the `CRM_Core_Form_EntityFormTrait` can assign its own tpl directly.

Reply:
Hmm - maybe - I think I would prefer to explore that as a follow up fix when I touch the next form